testdata/p4_16_samples_outputs/copy-frontend.p4

```diff
@@ -6,7 +6,7 @@ control c(inout bit<32> b) {
     S s1_0;
     S s2_0;
     @name("c.a") action a() {
-        s2_0 = { 32w0 };
+        s2_0 = {32w0};
```

Comment:
there should be no whitespace difference here. is this because you are missing a rebase?

Reply:
yes.